mods/species/tajaran/culture.dm

```diff
@@ -0,0 +1,48 @@
+//Culture
+
+/decl/cultural_info/culture/tajaran
+	name = CULTURE_TAJARA
+	description = "The Tajaran are a species of furred mammalian bipeds hailing from the chilly planet of Ahdomai \
```

Comment:
Would like to generalize this if possible, down to just 'they're a species of cat dudes from a frozen world'.